### ISSUE TITLE
UnhandledPromiseRejectionWarning - the bot stopped after a while

### ISSUE THREAD

**Issue by focamonca** [social-manager-tools/instagram-bot.js#84]
### Versions

Replace the values below with your own:

- **Bot Version:** v0.9.0
- **Node Version:** lastest
- **Operating System:** Raspbian 9
- **Browser:** chromium

### Expected Behavior

Please describe the program's expected behavior.

```
The bot should run over and over
```

### Actual Behavior

Please describe the program's actual behavior. Please include any stack traces
or log output in the back ticks below.

```
[INFO] like_realistic: try heart like
(node:5746) UnhandledPromiseRejectionWarning: Error: waiting for selector "article div a:nth-child(1)" failed: timeout 30000ms exceeded
    at Timeout.WaitTask._timeoutTimer.setTimeout (/home/pi/Desktop/instabotjs/node_modules/puppeteer/lib/FrameManager.js:844:60)
    at ontimeout (timers.js:498:11)
    at tryOnTimeout (timers.js:323:5)
    at Timer.listOnTimeout (timers.js:290:5)
(node:5746) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:5746) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.


```

### Steps to Reproduce

Please include the steps the reproduce the issue, numbered below. Include as
much detail as possible.

1. cd Desktop/instabotjs
2. node bot.js
3. wait some like

### config.js (remember to remove password)
```
module.exports = {
    // InstagramBot.js Configs
    "debug": true,
    "login": true,
    "ui": false, // only for social-manager-tools

    // Instagram Account
    "instagram_username": "", // without @
    "instagram_password": "",
    "instagram_hashtag": ["instagram", "muraleshunter", "hashtag"], //without #, raccomanded more than 20
    "instagram_pin": "sms", //method to receive pin (email or sms)

    // BOT Configs
    // [WORK] likemode_classic          - select random hashtag from config list and like 1 random photo (of last 20) | 400-600 like/day.
    // [WORK] likemode_realistic        - select random hashtag from config list, like fast 10-12 photo and sleep 15-20min. Sleep at night
    // [WORK] likemode_superlike        - select random hashtag from config list and like 3 random photo of same user | 400-600 like/day.
    // [WORK] likemode_competitor_users - select account, select random followers, like 10-12 photo and sleep 15-20min.
    // [WORK] comment_mode              - leave a comment under the photo via hashtag.
    // [WORK] fdfmode_classic           - follow 30 users, and defollow the first followed at 31 follow (in loop). This method is not detected from socialblade. ~1h | 300/300 follow-defollow/day.
    "bot_mode": "likemode_realistic",
    "bot_likeday_min": 800,      // work only in likemode_classic
    "bot_likeday_max": 900,      // work only in likemode_classic. In 2018 limit instagram is 1000/day
    "bot_sleep_night": false,    // if you need stop bot in night 
    "bot_start_sleep": "7:00",   // sleep from 00:00 to 7:00 am, work in likemode_realistic, comment_mode, likemode_superlike and likemode_competitor_users
    "bot_superlike_n": 3,        // superlike number of likes for user
    "bot_followday": 300,        // number of follow-defollow user/day, work in fdfmode_classic. In 2018 limit instagram is 500/day
    "bot_followrotate": 30,      // follow 30 users, and defollow the first followed at 31 follow (in loop), work in fdfmode_classic | If is 0 defollow all followed previously by bot
    "bot_userwhitelist": ["giulyegiammy", "powerslidelover", "ste_tami"],   // usernames without @ | don't defollow this users on follow-defollow mode, work in fdfmode_classic
    "comment_mode": {
        "comments": {
            "type": "array",
            "source": ["comment1", "comment2", "comment3"],
        }
    },
    "likemode_competitor_users": {
        "account": "user_nickname" // without @
    },

    // Puppeteer Configs
    "chrome_headless": true,
    "chrome_options": ["--disable-gpu", "--no-sandbox", "--window-size=1920x1080"],
    "executable_path": "/usr/bin/chromium-browser", // example for Mac OS: /Applications/Google\ Chrome.app/Contents/MacOS/Google\ Chrome

    // DATABASES
    "fdfdatabase_path": "./databases/fdf.db", // databases with username of followed users with follodefollow mode
    "logdb_path": "./databases/logs.db",      // logs in sqllite database for better report

    // LOG
    "pin_path": "./loginpin.txt",
    "log_path": "./logs/debug.log",
    "logerr_path":"./logs/errors.log",
    "screenshot_path": "./logs/screenshot/",
    "log": {
        "drivers": ["console"], // slack or console
        "screenshot": false,  // disable or enable screenshot in logs folder
        "channels": {
            "console": "",
            "slack": {
                "webhook": ""
            }
        }
    }
};


```

### Screenshots (Optional)

If the error is graphical in nature it is helpful to provide a screenshot. 

![image](https://user-images.githubusercontent.com/19846692/45477076-fde17000-b740-11e8-9ad0-e138748786b4.png)

**Comment by ptkdev** [comment]
User removed photo, https://www.instagram.com/p/BnqMV8lloBU/ don't exist. I know of this bug, i fix this in next release.

**Comment by focamonca** [comment]
but it did this error 3 times in row after restarting

**Comment by ptkdev** [comment]
@focamonca ok, instagram website have a problem. 2-3 persone write me of this error. Random instagram website not show hashtag pages or photos.

**Comment by ptkdev** [comment]
Fixed with **`v0.9.1`** release

<a href="http://patreon.ptkdev.io" alt="Patreon Backer"><img src="https://img.shields.io/badge/donate-patreon-F87668.svg"></a> <a href="http://paypal.ptkdev.io" alt="Paypale Donate"><img src="https://img.shields.io/badge/donate-paypal-46AFE0.svg"></a>  <a href="http://coffee.ptkdev.io" alt="Ko-fi Donate"><img src="https://img.shields.io/badge/buy%20me-coffee-4B788C.svg"></a>
